packages/devtools_app/lib/src/screens/memory/panes/diff/controller/heap_diff.dart

```diff
@@ -76,26 +76,15 @@ class HeapComparison {
       final youngerRecord = younger[key];
 
       if (olderRecord != null && youngerRecord != null) {
```

Comment:
now that the class has been renamed, should all vars with 'record' in their name be changed to 'stats'?